2021-10-06 23:57:23,911 INFO    MainThread:39534 [wandb_setup.py:_flush():69] setting env: {}
2021-10-06 23:57:23,911 INFO    MainThread:39534 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-06 23:57:23,911 INFO    MainThread:39534 [wandb_init.py:_log_setup():357] Logging user logs to /Users/kiranganeshan/reading-group/RL/wandb/run-20211006_235723-2ei3tau7/logs/debug.log
2021-10-06 23:57:23,911 INFO    MainThread:39534 [wandb_init.py:_log_setup():358] Logging internal logs to /Users/kiranganeshan/reading-group/RL/wandb/run-20211006_235723-2ei3tau7/logs/debug-internal.log
2021-10-06 23:57:23,912 INFO    MainThread:39534 [wandb_init.py:init():390] calling init triggers
2021-10-06 23:57:23,912 INFO    MainThread:39534 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-10-06 23:57:23,913 INFO    MainThread:39534 [wandb_init.py:init():435] starting backend
2021-10-06 23:57:23,913 INFO    MainThread:39534 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-06 23:57:23,927 INFO    MainThread:39534 [backend.py:ensure_launched():135] starting backend process...
2021-10-06 23:57:23,936 INFO    MainThread:39534 [backend.py:ensure_launched():140] started backend process with pid: 39561
2021-10-06 23:57:23,938 INFO    MainThread:39534 [wandb_init.py:init():440] backend started and connected
2021-10-06 23:57:23,943 INFO    MainThread:39534 [wandb_init.py:init():499] updated telemetry
2021-10-06 23:57:23,944 INFO    MainThread:39534 [wandb_init.py:init():522] communicating current version
2021-10-06 23:57:24,654 INFO    MainThread:39534 [wandb_init.py:init():527] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-06 23:57:24,655 INFO    MainThread:39534 [wandb_init.py:init():535] communicating run to backend with 30 second timeout
2021-10-06 23:57:24,796 INFO    MainThread:39534 [wandb_init.py:init():562] starting run threads in backend
2021-10-06 23:57:28,353 INFO    MainThread:39534 [wandb_run.py:_console_start():1651] atexit reg
2021-10-06 23:57:28,355 INFO    MainThread:39534 [wandb_run.py:_redirect():1525] redirect: SettingsConsole.REDIRECT
2021-10-06 23:57:28,356 INFO    MainThread:39534 [wandb_run.py:_redirect():1530] Redirecting console.
2021-10-06 23:57:28,358 INFO    MainThread:39534 [wandb_run.py:_redirect():1586] Redirects installed.
2021-10-06 23:57:28,358 INFO    MainThread:39534 [wandb_init.py:init():587] run started, returning control to user process
2021-10-06 23:57:28,427 INFO    MainThread:39534 [wandb_watch.py:watch():43] Watching
2021-10-06 23:57:31,718 INFO    MainThread:39534 [wandb_run.py:_atexit_cleanup():1621] got exitcode: 1
2021-10-06 23:57:31,719 INFO    MainThread:39534 [wandb_run.py:_restore():1593] restore
2021-10-06 23:57:34,265 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 689
}

2021-10-06 23:57:34,460 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 689
}

2021-10-06 23:57:34,845 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 246100
}

2021-10-06 23:57:34,950 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 246100
}

2021-10-06 23:57:35,053 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 246100
}

2021-10-06 23:57:35,159 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 226395
  total_bytes: 246100
}

2021-10-06 23:57:35,263 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 246100
  total_bytes: 246100
}

2021-10-06 23:57:35,369 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 246100
  total_bytes: 246100
}

2021-10-06 23:57:35,477 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 246100
  total_bytes: 246100
}

2021-10-06 23:57:35,584 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 246100
  total_bytes: 246100
}

2021-10-06 23:57:35,690 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 246100
  total_bytes: 246100
}

2021-10-06 23:57:35,998 INFO    MainThread:39534 [wandb_run.py:_wait_for_finish():1746] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 246100
  total_bytes: 246100
}
local_info {
}

2021-10-06 23:57:37,127 INFO    MainThread:39534 [wandb_run.py:_append_files():2009] logging synced files
